app/src/full/java/io/homeassistant/companion/android/sensors/AndroidAutoSensorManager.kt

```diff
@@ -25,29 +46,112 @@ class AndroidAutoSensorManager : SensorManager, Observer<Int> {
             "mdi:car",
             updateType = SensorManager.BasicSensor.UpdateType.INTENT
         )
+        private val fuelLevel = SensorManager.BasicSensor(
+            "android_auto_fuel_level",
+            "sensor",
+            commonR.string.basic_sensor_name_android_auto_fuel_level,
+            commonR.string.sensor_description_android_auto_fuel_level,
+            "mdi:barrel",
+            unitOfMeasurement = "%",
+            stateClass = SensorManager.STATE_CLASS_MEASUREMENT,
+            deviceClass = "battery"
+        )
+        private val batteryLevel = SensorManager.BasicSensor(
+            "android_auto_battery_level",
+            "sensor",
+            commonR.string.basic_sensor_name_android_auto_battery_level,
+            commonR.string.sensor_description_android_auto_battery_level,
+            "mdi:car-battery",
+            unitOfMeasurement = "%",
+            stateClass = SensorManager.STATE_CLASS_MEASUREMENT,
+            deviceClass = "battery"
```

Comment:
this should probably be set as diagnostic type for entity category to match the other battery level sensor we have

https://developers.home-assistant.io/blog/page/7?_highlight=diagnostic#entity-categories

May also want to look at the other sensors to see if they should be diagnostic or a primary sensor.